### PR TITLE
provision.docker: do not add route to container when HostPort is zero

### LINKED ISSUE #1280: containers-move: moving stopped container leaves wrong port 0 on router
When issue a "containers-move" from one machine to another, if any stopped containers are involved on process, tsuru will try to add port 0 on router for newly created container.

### PR DESCRIPTION
**Pull request by flaviamissi**
Fixes #1280

### CODE REVIEW

File: provision/docker/actions_test.go [hunk 1]
```diff
@@ -281,6 +281,29 @@ func (s *S) TestAddNewRouteForwardFailInMiddle(c *check.C) {
...
 
+func (s *S) TestAddNewRouteFowardDoesNotAddWhenHostPortIsZero(c *check.C) {
+	app := provisiontest.NewFakeApp("myapp", "python", 1)
+	routertest.FakeRouter.AddBackend(app.GetName())
+	defer routertest.FakeRouter.RemoveBackend(app.GetName())
+	cont := container.Container{ID: "ble-1", AppName: app.GetName(), ProcessName: "", HostAddr: "addr1", HostPort: "0"}
+	cont2 := container.Container{ID: "ble-2", AppName: app.GetName(), ProcessName: "", HostAddr: "addr2"}
+	defer cont.Remove(s.p)
+	defer cont2.Remove(s.p)
+
```
Comment (andrewsmedina) [inline provision/docker/actions_test.go]: can you remove this empty line?

Comment (flaviamissi) [inline provision/docker/actions_test.go]: done!

**Comment by andrewsmedina** [comment]
@flaviamissi thank you! can you also check if HotsPot is empty?

**Comment by flaviamissi** [comment]
good call @andrewsmedina, done!

**Comment by andrewsmedina** [comment]
@flaviamissi thank you!

**Comment by flaviamissi** [comment]
:metal: !

**Pull request merged**